src/utils/dialogmoteUtils.ts

```diff
@@ -0,0 +1,18 @@
+import { PersonOppgave } from "@/data/personoppgave/types/PersonOppgave";
+import {
+  DialogmoteDTO,
+  DialogmoteStatus,
+} from "@/data/dialogmote/types/dialogmoteTypes";
+
+export const isBehandletOppgave = (personOppgave: PersonOppgave): boolean => {
```

Comment:
Hører denne hjemme her? 🤔  Ser ikke ut til å ha noe med dialogmøte å gjøre.

Reply:
Godt poeng, jeg kan finne et nytt hjem for den 👪